torch/_torch_docs.py

```diff
@@ -13829,7 +13829,7 @@ def merge_dicts(*dicts):
 add_docstr(
     torch.expand_copy,
     r"""
-Performs the same operation as :func:`torch.expand`, but all output tensors
+Performs the same operation as :func:`torch.Tensor.expand`, but all output tensors
```

Comment:
Where is this? I don't see `torch.expand_copy` in the generated docs

Reply:
I suspect this would need to be added to https://github.com/pytorch/pytorch/blob/main/docs/source/tensors.rst?plain=1#L433 to show up

That said, seems like none of the `{*}_copy` functions (other than `index_copy` and `narrow_copy`) show up in documentation so it's probably ok not to add `expand_copy` there in this PR

@svekars do you know why all these `{*}_copy` variants are not in the generated docs and why [they are not in the coverage skiplist](https://github.com/pytorch/pytorch/blob/main/docs/source/conf.py?fbclid=IwZXh0bgNhZW0CMTEAAR0fbujOxNi9AmkYPYVPWNjmN36WcVVPWOk-ULliwfxjohC7ocLyHeYx5eg_aem_SSUJa4GtZyrmoeWPIEdAbA)?